packages/cerebral/src/Controller.js

```diff
@@ -24,13 +24,21 @@ class Controller extends EventEmitter {
     this.componentDependencyStore = new DependencyStore()
     this.options = options
     this.flush = this.flush.bind(this)
-    this.devtools = devtools
-    this.model = new Model({}, this.devtools)
+
+    // Only allow devtools in development
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
Hmm. Looks like it would break our demos. Because they are built with production environment set, but we still want it to support debugger.